lib/pdk/cli/util/changelog_generator.rb

```diff
@@ -0,0 +1,58 @@
+require 'bundler'
+
+module PDK
+  module CLI
+    module Util
+      class ChangelogGenerator
+        def generate_changelog
+          raise 'github_changelog_generator gem not installed' unless Bundler.rubygems.find_name('github_changelog_generator').any?
+          result = system('bundle exec rake changelog')
+
+          unless result
+            raise 'Error generating changelog'
+          end
+
+          output = PDK::Util::Filesystem.read_file('CHANGELOG.md')
+
+          raise 'Uncategorized PRs; Please go label them' if output =~ %r{UNCATEGORIZED PRS; GO LABEL THEM}
```

Comment:
This error message is not very communicative.  Which PRs? how do you label them? what label?

Reply:
Pointed the used to the generated file